packages/loopring_v3/contracts/impl/BasicDepositContract.sol

```diff
@@ -200,14 +199,12 @@ contract BasicDepositContract is IDepositContract, ReentrancyGuard
         )
         internal
     {
-        if (amount > 0) {
-            if (token == address(0)) {
-                // ETH
-                to.sendETHAndVerify(amount, gasleft());
-            } else {
-                // ERC20 token
-                token.safeTransferAndVerify(to, amount);
-            }
+        if (amount == 0) return;
```

Comment:
To be consistent I would add brackets here, because nowhere else I think we do this (I always use brackets).

Reply:
done